shared/AppInsightsCommon/src/Util.ts

```diff
@@ -479,53 +479,19 @@ export class Util {
     /**
      * generate random id string
      */
-    public static newId(): string {
-        const base64chars = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/';
-
-        let result = "";
-        // tslint:disable-next-line:insecure-random
-        let random = Math.random() * 1073741824; // 5 symbols in base64, almost maxint
-
-        while (random > 0) {
-            const char = base64chars.charAt(random % 64);
-            result += char;
-            random = Math.floor(random / 64);
-        }
-        return result;
-    }
+    public static newId = CoreUtils.newId;
 
     /**
      * generate a random 32bit number (-0x80000000..0x7FFFFFFF).
      */
     public static random32() {
-        return (0x100000000 * Math.random()) | 0;
+        return CoreUtils.random32(true);
```

Comment:
Moved the main implementation of these to CoreUtils, and added the ability to specify if you want signed or unsigned result, CoreUtils.random32(signed?: boolean). As this is a legacy version, this one will always returned a signed result as per documentation.